docs/opcodes/FF.mdx

```diff
@@ -6,8 +6,8 @@ group: System operations
 *Index 1 is top of the stack. See [PUSH](/#60).*
 
 ## Notes
-
-The current account is registered to be destroyed, and will be at the end of the current transaction. The transfer of the current balance to the given account cannot fail. In particular, the destination account code (if any) is not executed, or, if the account does not exist, the balance is still added to the given address.
+Post-Cancun behavior: send all Ether in the account to the target.
```

Comment:
This is not very clear. Maybe change to "From cancun", so that it's clear it's included in it